docs/en/integrations/language-clients/java/jdbc-v2.md

```diff
@@ -0,0 +1,141 @@
+---
+sidebar_label: JDBC V2
+sidebar_position: 4
+keywords: [clickhouse, java, jdbc, driver, integrate]
+description: ClickHouse JDBC driver V2
```

Comment:
> But I agree with this sentiment, it also might be worth us exploring at this point how do we do "versioning" of the documentation rather than having "extra pages" - like having a version drop down and corresponding version in the URL.

@laeg What about then having without these v1/v2 but using the client versioning? There will be 2 pages then
1. description: ClickHouse JDBC driver > v0.8.x (or ClickHouse JDBC driver latest)
2. description: ClickHouse JDBC driver v0.7.x

We can manually add more pages if necessary and they always transparently mapped to *JDBC release* version

Reply:
yeah i would prefer if we aligned it more to release versions rather than internal code versions. 

did we look into whether we could do something more elegant for versioning the docs?

Reply:
> did we look into whether we could do something more elegant for versioning the docs?

I didn't manage to find a solution in 30min.